src/reducers/project-text-search.js

```diff
@@ -53,6 +53,12 @@ function update(
       return state.merge({
         results: state.get("results").clear()
       });
+
+    case "CLOSE_PROJECT_SEARCH":
+      return state.merge({
+        query: "",
```

Comment:
@wldcordeiro should we use `state.remove("query")` instead? not sure why one is preferred over the other.